apps/angularlogos/src/app/logos-list/logos-list.component.ts

```diff
@@ -31,10 +33,14 @@ export class LogosListComponent implements OnInit {
       debounceTime(200),
       tap((term) => this.setSearchQueryParam(term)),
       startWith(''),
-      switchMap((term) => this.ds.getLogosFiltered(term))
+      switchMap((term) => this.dataService.getLogosFiltered(term))
     );
   }
 
+  searchTermChangeHandler(searchTerm: string) {
```

Comment:
I don't see the practical point of this refactoring. Can you elaborate?

Reply:
I always use real method on output emitters, more a habit kinda used to the pattern.